shared/cfg/guest-os/Windows.cfg

```diff
@@ -66,6 +66,47 @@
         migration_bg_kill_ignore_status = 128
     migrate.with_file_transfer:
         guest_path = C:\tmpfile
+    migrate_multi_host.with_stress, migrate_multi_host.set_downtime:
+        bg_stress_test = win_heavyload
+        timeout = 1800
+        autostress = yes
+        install_cmd = "start /wait DRIVE:\HeavyLoadSetup.exe /verysilent"
+        config_cmd = 'setx -m path "%PATH%;${install_path};"'
+        kill_bg_stress_cmd = "taskkill /T /F /IM heavyload.exe"
+        i386, i686:
+            install_path = "C:\Program Files\JAM Software\HeavyLoad"
+        x86_64:
+            install_path = "C:\Program Files (x86)\JAM Software\HeavyLoad"
+    migrate_multi_host.virtio_blk_data_plane:
+        bg_stress_test = fio_windows
+        timeout = 300
+        i386, i686:
+            install_path = "C:\Program Files (x86)\fio"
+            install_cmd = "msiexec /i DRIVE:\fio-2.2.10-x86.msi /qn"
+            #download_url = http://www.bluestop.org/fio/releases/fio-2.2.10-x64.msi
+            #pkg_md5sum = "fa45dee52e1f45e048e0cee2e14bdcd6"
+        x86_64:
+            install_path = "C:\Program Files\fio"
+            install_cmd = "msiexec /i DRIVE:\fio-2.2.10-x64.msi /qn"
+            #download_url = http://www.bluestop.org/fio/releases/fio-2.2.10-x86.msi
+            #pkg_md5sum = "ac10c6c33a3fa4475f162dbfc35005a3"
+        config_cmd = 'setx -m path "%PATH%;${install_path};"'
+        start_cmd = '"${install_path}\fio.exe" --name=fiotest --rw=randread --iodepth=4 --bs=4k --ioengine=windowsaio --size=1g --numjobs=4'
+        kill_bg_stress_cmd = "taskkill /T /F /IM fio.exe"
+    migrate_multi_host.with_module_reload:
+        driver_load_cmd = C:\devcon.exe enable @DRIVER_ID
+        driver_unload_cmd = C:\devcon.exe disable @DRIVER_ID
+        driver_id_cmd = C:\devcon.exe find * | find "VirtIO"
+        i386, i686:
+            devcon = "xcopy %s:\devcon\wnet_x86\devcon.exe C: /Y"
+        x86_64:
```

Comment:
x86_64  go into  guest-name/x86_64.cfg and move nic related cfg into guest-hw.cfg. thanks